src/test_doubly.py

```diff
@@ -0,0 +1,96 @@
+# _*_ encoding: utf-8 _*_
+import pytest
+"""Test linked_list.py."""
+
+INSERT_ITEMS = [(['one', 'two', 'three', 'four'], 4)]
+PREV_VAL = [(['one', 'two', 'three', 'four'], 'two')]
+POP = [(['one', 'two', 'three', 'four'])]
+APPEND = [(['one', 'two', 'three'], 'three')]
+SHIFT = [(['one', 'two', 'three', 'four'], 'four'),
+         (['five', 'hat', 'bunny', 'dude'], 'dude')]
+EMPTY = [([])]
+REMOVE = [(['one', 'two', 'three', 'four', 'five'], 'three', 'two', 'four', 4),
+          (['Hello', 'This', 'is', 'a', 'test'], 'a', 'is', 'test', 4)]
+
+
+def test_inheritance():
+    from doubly_linked import DoublyLinked
+    from linked_list import LinkedList
+    new_list = DoublyLinked()
+    assert isinstance(new_list, LinkedList)
```

Comment:
Doubly Linked Lists are different enough from Linked Lists that inheritance is not recommended here.